src/components/layouts/FilterBar.tsx

```diff
@@ -18,7 +31,60 @@ export const FilterBar = () => {
     >
       <AppMargin size={SizeofAppMargin.LARGE} py="18.5px">
         <Stack direction="row" alignItems="center" justifyContent="space-between">
-          <div> </div>
+          <Stack direction="row" columnGap={3}>
+            <Selector
+              getSelectedValue={(_newValue) => {
+                const newValue = _newValue as IAssigneeCombined
+                updateAssigneeValue(newValue)
+                store.dispatch(setFilteredAsignee(_newValue))
+              }}
+              startIcon={<FilterByAsigneeIcon />}
+              options={assignee}
+              placeholder="Assignee"
+              value={assigneeValue}
+              selectorType={SelectorType.ASSIGNEE_SELECTOR}
+              customOptions={{ id: '', name: 'No assignee', value: '' }}
+              buttonContent={
+                <Typography
+                  variant="bodySm"
+                  lineHeight="32px"
+                  fontWeight={500}
+                  fontSize="12px"
+                  sx={{ color: (theme) => theme.color.gray[600] }}
+                >
```

Comment:
Let's not use this `Typography` in a top level here. This `Typography` is rendered as `span` tag and technically we are rendering block elements inside the inline element which is not a preferred semantics. 
I think the node for `buttonContent` prop can be refactored for better readability as well. For ex: the way `<> Filter by </>` and `<> assignee </>` are rendered can be changed. Why not render both cases by breaking them down into smaller components that reside in the same file?